src/service/fixed-layer.js

```diff
@@ -586,16 +687,25 @@ export class FixedLayer {
    * @param {!Element} element
    * @param {string} selector
    * @param {string} position
-   * @param {boolean=} opt_forceTransfer If set to true , then the element needs
-   *    to be forcefully transferred to the transfer layer.
+   * @param {boolean=} opt_forceTransfer If true, then the element will
+   *    be forcibly transferred to the transfer layer.
+   * @param {boolean=} opt_lightboxMode If true, then descendants of lightboxes
+   *    are allowed to be set up. Default is false.
    * @private
    */
-  setupElement_(element, selector, position, opt_forceTransfer) {
+  setupElement_(
+    element, selector, position, opt_forceTransfer, opt_lightboxMode
```

Comment:
I think we should just setup everything, even descendants of lightbox.

Reply:
This avoids prematurely transferring lightbox descendants before the open animation completes.